Sources/Workspace/DefaultPluginScriptRunner.swift

```diff
@@ -444,14 +444,16 @@ public struct DefaultPluginScriptRunner: PluginScriptRunner, Cancellable {
         let stderrPipe = Pipe()
         let stderrLock = Lock()
         var stderrData = Data()
-        stderrPipe.fileHandleForReading.readabilityHandler = { fileHandle in
+        let stderrHandler = { (data: Data) in
             // Pass on any available data to the delegate.
-            stderrLock.withLock {
```

Comment:
Note that the lock isn't going away, it's being moved down to Line 456 since the `availableData`/`readToEnd()` has to be inside the lock and since it has to be moved out of the handler closure.